src/test/regress/sql/multi_size_queries.sql

```diff
@@ -0,0 +1,83 @@
+--
```

Comment:
Tests looks pretty good :+1:  Just remind me tomorrow to talk about restarting sequences at the beginning of test files.